Project.toml

```diff
@@ -22,7 +22,7 @@ JuMP = "~0.19.1"
 Memento = "~0.10, ~0.11, ~0.12"
 Missings = ">= 0.2.6"
 PowerModels = "~0.12"
-PowerSystemsUnits = "~0.1"
+PowerSystemsUnits = "~0.2"
```

Comment:
Should this be `PowerSystemsUnits = "~0.1, ~0.2"`?

Reply:
To use PowerSystemsUnits 0.1, it needs the line I deleted: https://github.com/lanl-ansi/PowerModelsAnnex.jl/pull/54/files#diff-c597fbb01271d69aecb55706e6905ea3L7

I would not recommend continuing to use 0.1. We know it has issues which can cause segfaults.

Reply:
Ok.  Sounds good then.